src/pages/cart/Cart.js

```diff
@@ -2,84 +2,91 @@ import { NavBar } from "../../components/NavBar/NavBar";
 import "./cart.css";
 import sofa from "../../Assets/images/sofa.jpeg";
```

Comment:
export all your images in an index file and import images from it. So you don't have to write a relative path every time.